tests/unit/test_table_options.py

```diff
@@ -132,83 +188,89 @@ def test_table_require_partition_filter_dialect_option(faux_conn):
             bigquery_require_partition_filter=True,
         )
 
-    assert " ".join(faux_conn.test_data["execute"][-1][0].strip().split()) == (
+    result = " ".join(faux_conn.test_data["execute"][-1][0].strip().split())
```

Comment:
NIT:

It seems this test is covered in `test_table_time_partitioning_date_timestamp_and_datetime_dialect_option`?

Reply:
Deduplicated.